src/main/java/uk/gov/hmcts/reform/sscs/model/SessionCaseCodeMapping.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder(toBuilder = true)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SessionCaseCodeMapping {
```

Comment:
Can we get a method for constructing the category value, e.g. if benefitCode = 001, issueCode = AD, ServiceCode = BBA3, then the category value is BBA3-001AD

Reply:
something like:
`        public static final String SERVICE_CODE = "BBA3";

        String getCategoryTypeValue() {
            return String.format("%s-%03d",SERVICE_CODE, benefitCode);
        }

        String getCategorySubTypeValue() {
            return String.format("%s%s",getCategoryTypeValue(),issueCode);
        }`